discord_bot.py

```diff
@@ -1,17 +1,9 @@
 import logging
-import os
 
-import discord
 from discord.ext import commands
 
-BASE_PATH = os.path.dirname(os.path.abspath(__file__))
-MEDIA_PATH = os.path.join(BASE_PATH, 'media/')
-if not os.path.exists(MEDIA_PATH):
-    os.makedirs(MEDIA_PATH)
 
 class HuskieBot(commands.Bot):
-    def __init__(self, *args, **kwargs):
```

Comment:
See, this is what I was talking about what we need to do for the cogs, get rid of the `__int__`'s where where don't do anything besides the default and just let it call `super().__init__(*args, **kwargs)` for us behind the scenes

Reply:
After checking the source code for cogs, we saw that cogs don't actually have a default `__init__` function, so this whole comment is moot